job_definitions/clean_and_migrate_db_dump.yml

```diff
@@ -0,0 +1,101 @@
+- job:
+    name: clean-and-migrate-db-dump
+    display-name: Clean and migrate database dump
+    project-type: pipeline
+    description: Takes the latest production database dump, cleans it, and migrates to target stage. Also Google Drive.
+    concurrent: false
+    triggers:
+      - timed: "0 4 * * 0"
+    parameters:
+      - extended-choice:
+          name: TARGET_STAGE
+          type: radio
+          default-value: staging
```

Comment:
I'm going to leave it as staging as we don't have another way to schedule a parameterised job, without creating a another job.